devito/passes/__init__.py

```diff
@@ -1,3 +1,40 @@
+# The following functions may be used by passes at any layer of abstraction
+
+from devito.logger import warning
+from devito.types.dense import AliasFunction, TimeFunction
+from devito.tools import as_tuple, is_integer
+
+
+def is_on_device(obj, gpu_fit):
```

Comment:
Not super fan of a function inbetween imports.  Can this be moved in its own "utils" file to make it cleaner?

Reply:
I don't see anything odd in keeping it there, an extra folder + file + __init__ sounds like an overkill also